crates/bevy_core_pipeline/src/oit/mod.rs

```diff
@@ -90,17 +105,32 @@ impl Plugin for OrderIndependentTransparencyPlugin {
 
 // WARN This should only happen for cameras with the [`OrderIndependentTransparencySettings`]
 // but when multiple cameras are present on the same window
-// bevy reuses the same depth texture so we need to set this on all cameras.
+// bevy reuses the same depth texture so we need to set this on all cameras with the same render target.
 fn configure_depth_texture_usages(
-    mut new_cameras: Query<
-        &mut Camera3d,
-        (Added<Camera3d>, With<OrderIndependentTransparencySettings>),
-    >,
+    p: Query<Entity, With<PrimaryWindow>>,
+    cameras: Query<(&Camera, Has<OrderIndependentTransparencySettings>)>,
+    mut new_cameras: Query<(&mut Camera3d, &Camera), Added<Camera3d>>,
 ) {
-    for mut camera in &mut new_cameras {
-        let mut usages = TextureUsages::from(camera.depth_texture_usages);
-        usages |= TextureUsages::RENDER_ATTACHMENT | TextureUsages::TEXTURE_BINDING;
-        camera.depth_texture_usages = usages.into();
+    if new_cameras.is_empty() {
+        return;
+    }
+
+    // Find all the render target that potentially uses OIT
+    let primary_window = p.get_single().expect("No primary window found");
```

Comment:
I don't think we want to panic if there's not a primary window, since that's a valid way to build an app. This should be changed to 

```rust
let primary_window = p.get_single().ok();
```
and then passed as is to the normalize calls.